core/account-id/README.md

```diff
@@ -1,69 +1,36 @@
 # near-account-id
 
-This crate provides a type for representing a valid, unique account identifier on the [NEAR](https://near.org) network.
+This crate provides a type for representing a syntactically valid, unique account identifier on the [NEAR](https://near.org) network.
 
 [![crates.io](https://img.shields.io/crates/v/near-account-id?label=latest)](https://crates.io/crates/near-account-id)
 [![Documentation](https://docs.rs/near-account-id/badge.svg)](https://docs.rs/near-account-id)
-[![Version](https://img.shields.io/badge/rustc-1.56+-ab6000.svg)](https://blog.rust-lang.org/2021/10/21/Rust-1.56.0.html)
-[![Apache 2.0 licensed](https://img.shields.io/crates/l/near-account-id.svg)](https://github.com/near/nearcore/blob/master/licenses/LICENSE-APACHE)
+![MIT or Apache 2.0 licensed](https://img.shields.io/crates/l/near-account-id.svg)
 
 ## Usage
 
 ```rust
 use near_account_id::AccountId;
 
-let alice: AccountId = "alice.near".parse().unwrap();
+let alice: AccountId = "alice.near".parse()?;
 
-// Basic reports for why validation failed
-assert!(
-  matches!(
-    "z".parse::<AccountId>(),
-    Err(err) if err.kind().is_too_short()
-  )
-);
+assert!("ƒelicia.near".parse::<AccountId>().is_err()); // (ƒ is not f)
+```
 
-assert!(
-  matches!(
-    // no caps
-    "MelissaCarver.near".parse::<AccountId>(),
-    Err(err) if err.kind().is_invalid()
-  )
-);
+See the [docs](https://docs.rs/near-account-id) for more information.
 
-assert!(
-  matches!(
-    // separators cannot immediately follow each other
-    "bob__carol".parse::<AccountId>(),
-    Err(err) if err.kind().is_invalid()
-  )
-);
+## License
 
-assert!(
-  matches!(
-    // each part must be alphanumeric only (ƒ is not f)
-    "ƒelicia.near".parse::<AccountId>(),
-    Err(err) if err.kind().is_invalid()
-  )
-);
-```
+Licensed under either of
+
+- Apache License, Version 2.0
+   ([LICENSE-APACHE](LICENSE-APACHE) or <http://www.apache.org/licenses/LICENSE-2.0>)
+- MIT license
+   ([LICENSE-MIT](LICENSE-MIT) or <http://opensource.org/licenses/MIT>)
 
-## Account ID Rules
+at your option.
```

Comment:
I'm surprised that AccountId rules disappeared from this documentation

Reply:
discussed offline: please add the link to the account rules: https://docs.near.org/docs/concepts/account#account-id-rules